windows/EFwebConnector.bat

```diff
@@ -47,8 +49,17 @@ REM     Then the information for joining the chosen server is extracted from the
 REM     and put in a format that the Elite Force executable can understand
 
 SET ARGUMENTS=%1
-SET "ARGUMENTS=%ARGUMENTS:stvef:=%"
-FOR /f "tokens=1,2,3,4 delims=; " %%a IN ("%ARGUMENTS%") DO SET "SERVER_IP=%%a"&SET "SERVER_PORT=%%b"&SET "SERVER_NAME=%%c"&SET "MAP_NAME=%%d"
+SET ARGUMENTS=%ARGUMENTS:stvef:=%
+SET ARGUMENTS=%ARGUMENTS:"=%

```

Comment:
This line removes any `"` present in the arguments. I think the way the protocol handler is registered is causing an extra `"` at the beginning and end of the argument, which was causing some issues below.